src/mem/cache/Cache.py

```diff
@@ -111,6 +111,12 @@ class BaseCache(ClockedObject):
     replacement_policy = Param.BaseReplacementPolicy(
         LRURP(), "Replacement policy"
     )
+    partitioning_policies = VectorParam.BasePartitioningPolicy(
+        [],
+        "Partitioning policies"
+        "Setting multiple policies will enforce all of them individually"
+        "in order",
```

Comment:
```suggestion
        "Partitioning policies "
        "Setting multiple policies will enforce all of them individually "
        "in order",
```

Tiny thing, but there needs to be spaces or else it will appear as "Partitioning policiesSetting..."

Reply:
Fixed in [commit b28541d](https://github.com/gem5/gem5/pull/765/commits/b28541dda3b173d4a2829b93a680c6cc3fa4d384)